rpki-validator/src/main/java/net/ripe/rpki/validator3/background/ValidationScheduler.java

```diff
@@ -42,28 +45,54 @@
 import org.quartz.TriggerBuilder;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.stereotype.Component;
 
 import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.TemporalField;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 
 @Component
 @Slf4j
 public class ValidationScheduler {
 
     private final Scheduler scheduler;
+
     @Getter
     private final Duration rsyncRepositoryDownloadInterval;
+    @Getter
     private final Duration rrpdRepositoryDownloadInterval;
+
+    private final Throttled<String> throttledTreeValidation;
+
     private boolean enabled = true;
 
+    private final CertificateTreeValidationService validationService;
+
     @Autowired
     public ValidationScheduler(Scheduler scheduler,
                                @Value("${rpki.validator.rsync.repository.download.interval:PT10M}") String rsyncRepositoryDownloadInterval,
```

Comment:
Not a change, but: `@Value(...) Duration argName` would be possible iirc

Reply:
Makes sense, but I would do it separately as we have multiple places with these intervals.